tests/test_kubernetes.py

```diff
@@ -201,4 +201,44 @@ def _ensure_that_proxied_service_is_deleted():
     if 'client-custom' in [
         svc['metadata']['name'] for svc in kubectl.json(["-n", "server", "get", "svc"])["items"]
     ]:
-        pytest.skip("The orphaned service should be removed, but it's not critical, so skipping for now")
+        pytest.skip(
+            "The orphaned service should be removed, but it's not critical, so skipping for now"
+        )
+
+
+def test_connection_via_the_tunnel(
+    kubectl,
+    k8s_server,
+    k8s_client,
+    mock_server,
+):
+
+    annotator = Annotator(mock_server, kubectl)
+    amount_of_services_before_annotation = len(
+        kubectl.json(["-n", "server", "get", "svc"])["items"]
+    )
+    annotator.do("wormhole.glothriel.github.com/exposed", "yes")
+
+    @retry(tries=60, delay=1)
+    def _ensure_that_proxied_service_is_created():
+        assert (
+            len(kubectl.json(["-n", "server", "get", "svc"])["items"])
+            == amount_of_services_before_annotation + 1
+        )
+    _ensure_that_proxied_service_is_created()
```

Comment:
Looks like a candidate another fixture/helper - you have the same thingie in `test_changing_annotation_causes_creating_proxy_service`

`expose_service(annotator)`

Reply:
I added a `Service` util with a classmethod for getting amount of services in given ns and list of service names. I don't want the tests to hide too much, they should be verbose enough so you immediately know what the test is for.